pyproject.toml

```
@@ -33,6 +33,7 @@ requests_unixsocket = "^0.2.0"
 protobuf = "^3.12.2"
 namecom = "^0.5.0"
 crontab = "^0.22.9"
+python-telegram-bot = ""
```

we should add version, 

```suggestion
python-telegram-bot = "^0.0.1"
```

and should be added using in order to update `poetry.lock` file aswell

```
poetry add telegram
```